lib/Entrypoint.js

```diff
@@ -41,6 +41,9 @@ Entrypoint.prototype.getSize = function(compilation) {
 	var files = this.getFiles();
 
 	return files
+		.filter(function(asset) {
+			return !(/\.map$/.test(asset))
+		})
```

Comment:
Does asset have a Class yet? Maybe this is where this stuff should end up as a refactor point.